components/Toolbar/Toolbar.jsx

```diff
@@ -130,22 +130,18 @@ export const Toolbar = () => {
                 className={styles.Toolbar__container}
             >
                 {!isOpenSidePage ? (
-                    <>
+                    <div>
```

Comment:
**question** why needed replace React.Fragment -> div ?

Reply:
@AtkishkinVlad we have two icons in this case

Reply:
I can see it, but it seems like layout will be cleaner without div

Reply:
So far, it was important for me to fix the problem with the order of icons.

In the next PR, it will be great if we not only fix the width, but also add the loader and icons. But we will also combine the components `<Toolbar>` and `<MainPageToolbar>` :)